lib/sles4sap/sap_deployment_automation_framework/configure_tfvars.pm

```diff
@@ -116,34 +125,85 @@ sub set_workload_vnet_name {
     set_var('SDAF_SUT_VNET_NAME', 'OpenQA-' . $args{job_id});
 }
 
-=head2 set_vm_image_parameters
+=head2 set_image_parameters
 
-    set_vm_db_image_parameters([job_id=>'123456']);
-
-=over
-
-=item * B<$job_id>: Specify job id to be used. Default: current deployment job ID
```

Comment:
This was not even required for original function. Some copy paste slipped there :D